### PR TITLE
Fixes in GetV6BridgeID & SendV6Command with timing and unexpected mes…

### PR DESCRIPTION
**Pull request by chrisverhoef**
Add some fixes for people who also uses for example the andriod app. Multiple sources to the ibox let the ibox send some undocumented comments. I skip them for now and ask LimitlessLed what the command means and how to act.

### CODE REVIEW

File: hardware/Limitless.cpp [hunk 1]
```diff
@@ -238,15 +238,17 @@ bool CLimitLess::StartHardware()
 				{
 					if (!AddSwitchIfNotExits(3, "AppLamp Group3"))
 					{
-						AddSwitchIfNotExits(4, "AppLamp Group4");
+						if (!AddSwitchIfNotExits(4, "AppLamp Group4"))
+						{
+							if (m_BridgeType == LBTYPE_V6)
```
Comment (gizmocuz) [inline hardware/Limitless.cpp]: This was removed for a reason.
If you switch from bridge types and do not change the light type, the 5 lamp groups still exist, but the bridge light never gets created
Leaving in where it was before does not do any harm

**Review by gizmocuz — COMMENTED** [review]
Thanks, but please undo one change

File: hardware/Limitless.cpp (hunk 1 above)
Comment (chrisverhoef) [reply hardware/Limitless.cpp]: Will do, this was not a change on purpose but from a behind fork. Will send new pull request later.

Comment (gizmocuz) [reply hardware/Limitless.cpp]: if you (keep) push changes to your fork, they will be automatically here

**Comment by gizmocuz** [comment]
Thanks!

**Pull request merged**